protos/google/api/client.proto

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Google LLC.
```

Comment:
minor: `s/2019/2020`

Reply:
The proto files are direct copy-paste from the googleapis GitHub project. I'd rather not make any changes to them.